src/lib/Tab/Dashboard/PagerContentToDataMapper.php

```diff
@@ -18,6 +18,10 @@
 use EzSystems\EzPlatformAdminUi\Pagination\Mapper\AbstractPagerContentToDataMapper;
 use Pagerfanta\Pagerfanta;
 
+/**
```

Comment:
You should also deprecate service definition + throw `E_DEPRECATED` error or no one will really notice that.